extensions/stackdriver/log/logger.cc

```diff
@@ -48,10 +49,18 @@ Logger::Logger(const ::wasm::common::NodeInfo& local_node_info,
   log_entries_request_->set_log_name("projects/" + project_id + "/logs/" +
                                      kServerAccessLogName);
 
+  std::string resource_type = Common::kContainerMonitoredResource;
```

Comment:
nit: s/std::string/auto&. Save a copy of string.

Reply:
done.